zebra-rpc/src/methods.rs

```diff
@@ -1938,3 +1751,12 @@ pub fn height_from_signed_int(index: i32, tip_height: Height) -> Result<Height>
         Ok(Height(sanitized_height))
     }
 }
+
+/// Turns anything implementing [`ToString`] into [`Error`].
+fn to_error(err: impl ToString) -> Error {
+    Error {
+        code: ErrorCode::ServerError(0),
+        message: err.to_string(),
+        data: None,
+    }
+}
```

Comment:
Could we replace all uses of this with `ok_or_server_error()`?

```suggestion
```

Reply:
Done in [2c3a8db](https://github.com/ZcashFoundation/zebra/pull/8533/commits/2c3a8dbe8f3dd4a5bb256829c2e44f993d8d32de).